invenio.cfg

```diff
@@ -567,4 +567,10 @@ USERS_RESOURCES_ADMINISTRATION_ENABLED = True
 """Enable user administration."""
 
 COMMUNITIES_ADMINISTRATION_DISABLED = False
-"""Enable communities administration."""
\ No newline at end of file
+"""Enable communities administration."""
+
+MAIL_DEFAULT_SENDER = "Zenodo <info@zenodo.org>"
+"""Default e-mail address sender."
+
+REST_ENABLE_CORS = True
+"""Enable CORS validation.""""
```

Comment:
Should we evaluate the [CSP rules](https://github.com/zenodo/zenodo-rdm/pull/542/files#diff-35ea7d0c9f0d3771b33ec0ac671404047bdff444801fcf084c39437c21107ebfR85-R86) as well? Especially, the fact that we allow `unsafe-inline` and `unsafe-eval`?